frontend/cypress/support/commands.ts

```diff
@@ -0,0 +1,41 @@
+declare namespace Cypress {
+  interface Chainable {
+    playEvents(num: number): Chainable<Response>;
+    checkEventPresence(): Chainable<Response>;
+    playUntilDate(date: string): Chainable<Response>;
+  }
+}
+
+Cypress.Commands.add('playEvents', (num) => {
+  let i = 0;
+  for (i; i<num; i++) {
+    cy.get('[data-cy=event]', {timeout: 2000}).then(() => {
+      cy.wait(500);
+      cy.get('[data-cy=event-continue]', {timeout: 2000}).first().click();
+    });
+    cy.wait(500);
+  }
+})
+
+Cypress.Commands.add('checkEventPresence', () => {
+  cy.get('cvd-events-layout').then(($events) => {
+    if ($events.children().length > 0) {
+      cy.playEvents(1);
+    }
+  });
+})
+
+Cypress.Commands.add('playUntilDate', (date) => {
+  // Requires medium speed and then plays for a month
```

Comment:
Why is the medium speed required? Is it bcs of all the `cy.wait(..)` ?

Reply:
yeah... there is a step of wait for 1000.. which corresponds to the medium speed tick... otherwise it't wouldn't need just 35 iterations in the loop... but more/less... and if less then there is a chance it would jump over the date we are looking for